kinitro/scheduler/scoring.py

```
@@ -0,0 +1,157 @@
+        env_ids: List of environment IDs
+        episodes_per_env: Number of episodes per environment
+        pareto_temperature: Softmax temperature for weight conversion
+
+    Returns:
+        Tuple of (weights_dict, weights_u16_dict)
+
+    Raises:
+        ValueError: If pareto_temperature is not positive
+    """
+    if pareto_temperature <= 0:
+        raise ValueError("pareto_temperature must be > 0")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)